usagov/vote-gov
# VOTE-432 -> update of cypress tests for new homepage format

Link to ticket -> https://bixal-projects.atlassian.net/browse/VOTE-432

Purpose or PR -> with the update of the new homepage format this added a separate test to check the accordions on the page and ensure they are clickable and that there is text present once the accordion is opened.  Also, the test to check the links on the homepage was redone.  Before there was an error because there were multiple links that were the same.  now the test will check for any link on the page and that it has a status of 200.. this will help for any other future links that are added or repeated.

### to test:

- pull down the branch 
- run `npm run start`
- spilt terminal 
- run `npm run cy:open` and once the cypress runner opens click the home page test and verify that tests are passing
- can also check the GitHub action of the `cypress functional` testing suit to verify that tests are passing
